docs/en/reference/configuration.rst

```diff
@@ -312,6 +312,7 @@ pdo_oci / oci8
    connection pooling.
 -  ``charset`` (string): The charset used when connecting to the
    database.
+
```

Comment:
Please remove the extra line you've introduced.

Reply:
Done